framework/integration/wrapper/configmap/wrapper.go

```diff
@@ -1,66 +1,62 @@
 // +build k8srequired
 
-package client
+package configmap
 
 import (
 	"time"
 
-	"github.com/giantswarm/e2e-harness/pkg/harness"
-	"github.com/giantswarm/microerror"
-	"github.com/giantswarm/micrologger"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
 
+	"github.com/giantswarm/e2e-harness/pkg/harness"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
 	"github.com/giantswarm/operatorkit/framework"
 	"github.com/giantswarm/operatorkit/framework/integration/testresourceset"
 	"github.com/giantswarm/operatorkit/informer"
 )
 
-var (
-	err error
+type Wrapper struct {
+	framework *framework.Framework
 
 	k8sClient kubernetes.Interface
-)
+}
 
 type Config struct {
```

Comment:
We have the configs first and then the objects. Then the constructors and so on.